tox.ini

```diff
@@ -1,16 +1,19 @@
 [tox]
-envlist = py3{5,6,7,8,9}
+envlist = py3{6,7,8,9,10}-{windows,linux}
```

Comment:
Indeed, it is a good idea to keep 3.5 running. At my workplace, we recently settled on using 3.5 instead of using a mix of 3.1 and 3.5 . Old python versions still have a long life in front of them.